src/main/java/com/hedvig/backoffice/graphql/resolvers/QuestionGroupResolver.kt

```diff
@@ -0,0 +1,50 @@
+package com.hedvig.backoffice.graphql.resolvers
+
+import com.coxautodev.graphql.tools.GraphQLResolver
+import com.hedvig.backoffice.graphql.GraphQLConfiguration
+import com.hedvig.backoffice.graphql.types.Member
+import com.hedvig.backoffice.graphql.types.Member.Companion.fromDTO
+import com.hedvig.backoffice.graphql.types.questions.QuestionGroupType
+import com.hedvig.backoffice.services.members.MemberService
+import com.hedvig.backoffice.services.personnel.PersonnelService
+import com.hedvig.backoffice.util.Gender
+import graphql.schema.DataFetchingEnvironment
+import org.springframework.stereotype.Component
+import java.time.Instant
+
+@Component
+class QuestionGroupResolver
+  (
+  private val memberService: MemberService,
+  private val personnelService: PersonnelService
+) :
+  GraphQLResolver<QuestionGroupType?> {
+  fun getMember(
+    QuestionGroup: QuestionGroupType,
+    env: DataFetchingEnvironment?
+  ): Member {
+    return try {
+      fromDTO(
+        memberService.findByMemberId(
+          QuestionGroup.memberId,
+          GraphQLConfiguration.getIdToken(env, personnelService)
+        )
+      )
+    } catch (e: Exception) {
+      Member(
+        QuestionGroup.memberId,
+        "UNKNOWN",
+        "UNKNOWN",
+        "Unknown",
+        Gender.OTHER,
+        "Unknown",
+        "Unknown",
+        "Unknown",
+        Instant.now(),
+        "Unknown",
+        "Unknown"
+      )
+    }
```

Comment:
The try catch is unnecessary here since it is will most likely never happen:
```suggestion
      fromDTO(
        memberService.findByMemberId(
          QuestionGroup.memberId,
          GraphQLConfiguration.getIdToken(env, personnelService)
        )
      )
```

Reply:
Also, for code clartiy, we should not import static companion functions like this:
`fromDTO` should be `Member.fromDTO`

Reply:
It does cause a exception-crash without it, on the data I currently have. Are we sure its not just best to keep it in for those few times it does happen?

Reply:
That's because you are stubbing member-service. With correct data accross the services that should not happen. You could also fix the member-service stub to not return nothing if a member is not found.